service/amf_server.go

```diff
@@ -122,10 +123,11 @@ func (s *Server) HandleMessage(srv sdcoreAmfServer.NgapService_HandleMessageServ
 	return nil
 }
 
-func StartGrpcServer(port int) {
+func StartGrpcServer(ctx context.Context, port int) {
 	endpt := fmt.Sprintf(":%d", port)
 	fmt.Println("listen - ", endpt)
```

Comment:
I think we should change this to a "regular" log

```suggestion
	logger.GrpcLog.Infof("AMF gRPC server is starting on port %s", endpt)
```